client/components/Add-Answer.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
```

Comment:
Can we please change the name of this file to `AddAnswer.jsx` (without the hyphen)?